src/scss/src/foundation/_variables.scss

```diff
@@ -42,11 +42,133 @@
 
 /* Font families */
 
-/* Font weights */
+$rubik: 'Rubik, -apple-system, Arial, sans-serif';
 
-/* Sizes */
+/* Font weights */
 
-/* Heading sizes */
+$fw-regular: 400;
+$fw-medium: 500;
+$fw-semibold: 600;
+$fw-bold: 800;
+
+$fw: (
+  regular: $fw-regular,
+  medium: $fw-medium,
+  semibold: $fw-semibold,
+  bold: $fw-bold
+);
+
+/* Font sizes (headings) */
+
+$fs-h1: 4.5rem; // 72px
+$fs-h2: 3.813rem; // 61px
```

Comment:
why you don`t set it to px instead of rem?

Reply:
This is because nowadays, screen sizes come in different sizes and shapes. If we use `px`, the element's size remains constant regardless of the size of the screen. So using relative units like `em` and `rem` are considered good practice.

Also we are following design system from this senior course: https://softserveacademy.udemy.com/course/react-for-senior-engineers/learn/lecture/26838016#overview. Rem is also used here

I added pixels in comments for us, developers, to make our life easier and to avoid breaking our minds calculating value in pixels :)

Reply:
got it